cms/templates/container.html

```diff
@@ -144,7 +149,9 @@ <h3 class="sr">${_("Page Actions")}</h3>
                 % endif
             </ul>
         </nav>
+    % if is_unit_page:
```

Comment:
Nit: might be helpful here to explain why this is sometimes hidden, i.e. add a comment that says

```mako
## Hide the sequence navigation when we've browsed into a child of the unit, e.g. showing the child blocks of a problem-builder xblock
```

Reply:
Comment Added